tests/test_cli.py

```diff
@@ -58,8 +66,17 @@ def test_reduce_union():
     with open("tests/data/trio.seq") as seq:
         data = seq.read()
 
+    # Define our reduce command using a mkdocs snippet.
+    arg = """
+    --8<-- [start:reduce]
+    unary_union c
+    --8<-- [end:reduce]
+    """.splitlines()[
+        2
+    ].strip()
```

Comment:
@kevinlacaille I've got some tested CLI snippets! It's a little tortured, I'm working against Python to keep quotes out of the snippet section, but it works!

![Screenshot from 2023-05-03 15-38-14](https://user-images.githubusercontent.com/33697/236055705-cfc6b34f-21e3-4015-a298-d3be3b76ccba.png)